src/index.ts

```diff
@@ -111,7 +111,7 @@ function parseProfileId(hexx: string): string {
   hex = hex.slice(2);
   var str = "";
   for (var i = 0; i < hex.length; i += 2) {
-    const ch = String.fromCharCode(parseInt(hex.substring(i, 2), 16));
+    const ch = String.fromCharCode(parseInt(hex.substr(i, 2), 16));
```

Comment:
Isn't it a deprecated function? Btw, let's hide the "helper functions" in somewhere else, not inside the index.ts file to make it clearer.

Reply:
`hex.substring(i, 3)`